### ISSUE TITLE
Function to expand symbol-separated fields

### ISSUE THREAD

**Issue by calebasaraba** [ecohealthalliance/eidith#64]
Many P2 data fields are semicolon- or comma-separated strings representing multiple-selection responses to particular questions. The large number of the options per field makes universal expansion of them impractical, but providing a quick function to expand a column of interest would be very helpful for beginning r users of the package.

**Comment by noamross** [comment]
So this would be based on `tidyr::separate`, and change `animals_present to things like `animals_present_dogs`, `animals_present_cats`?  Or would it make sense for the function to convert the delimited column to a list-column?

**Comment by calebasaraba** [comment]
I think the former makes the most sense -- came up in talking with P2 data user who struggled to deal with the current format. I think a list-column would be more challenging for the early user?

**Comment by noamross** [comment]
Agreed.  If the function is less than two or three lines, though, we may want to make it a "recipe" in the tutorials.
